src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher1.php

```diff
@@ -36,36 +36,38 @@ public function match($pathinfo)
 
 
         // foo
-        if (0 === strpos($pathinfo, '/foo') && preg_match('#^/foo/(?P<bar>baz|symfony)$#s', $pathinfo, $matches)) {
+        if (0 === strpos($pathinfo, '/foo/') && preg_match('#^/foo/(?P<bar>baz|symfony)$#s', $pathinfo, $matches)) {
```

Comment:
Are you sure that adding the `/` is a good idea? It forbids the next routes to be merged together via the `/bar` prefix. Not sure if that's something that actually happens a lot in real-life though.